packaging/gimp-fourier-plugin.spec.in

```diff
@@ -0,0 +1,55 @@
+#
+# spec file for package gimp-fourier-plugin
+#
+# Copyright (c) 2011 Kyrill Detinov (Version 0.4.1)
+# This file and all modifications and additions to the pristine
+# package are under the same license as the package itself.
+# Modified for autoconf style build by Joe Da Silva (v0.4.3)
+#
+Name:		fourier
+Version:	@FOURIER_VERSION@
+Release:	0
+Summary:	Do direct and reverse Fourier Transforms on your image
+License:	GPLv3+
+URL:		https://www.lprp.fr/gimp_plugin_en/
+Group:          Productivity/Graphics/Bitmap Editors
+Source0:	https://github.com/rpeyron/plugin-gimp-fourier/releases/download/v%{version}/fourier-%{version}.tar.gz
+BuildRequires:	autoconf
+BuildRequires:	automake
+BuildRequires:	gcc
+BuildRequires:	make
+BuildRequires:  pkgconfig(fftw3)
+BuildRequires:  pkgconfig(gimp-2.0)
+Requires:       gimp
+Requires:       libfftw3
+
+%description
+A simple plug-in to do fourier transform on your image. The major advantage
+of this plugin is to be able to work with the transformed image inside GIMP.
+You can draw or apply filters in fourier space and get the modified image
+with an inverse fourier transform.
+Useful in fixing moire patterns or fixing some regular banding noise.
+
+%prep
+%setup ‑q
+
+%build
+#%autoreconf -i
+#%automake
+%configure --bindir=%{_libdir}/gimp/2.0/plug-ins
+%make_build
+
+%install
+# We can’t use “gimptool-2.0 install-bin” to install into plug-ins directory.
```

Comment:
I am a little curious why not to use gimptool-2.0 --install-admin-bin to install files system-wide ?

Reply:
I tried using gimptool. It does work, and it will install a personal copy in a hidden folder on the user's home/user directory. However, even though it was there, GIMP did not show the FFT Forward/Inverse filter choices, so I suspect that GIMP hasn't been debugged to make use of this (I believe this method could work for corporate users who cannot install as root - but seeing it didn't find fourier, I suspect more work is needed in GIMP - if you want to test it in linux, just run 'make install-user' or 'make uninstall-user'). The majority of linux users are home users, and they use the distro provided install/remove method, this also tends to be the method of install for shared software for multiple users in a linux environment. Testing 'sudo make install' showed that it was installed in the 2.0/plug-ins/ directory and GIMP also found fourier since I could see menu choices FFT Forwards/Inverse).
I am guessing gimptool also installs in a user home directory on windows too (like with linux), and therefore each user would have to login and install their own copies

Reply:
> # We can’t use “gimptool-2.0 install-bin” to install

Is actually a comment found in the fedora RPM spec file.
I tried the gimptool, it installs, but GIMP did not appear to find the installed fourier.
The shared plugin directory is probably the better way to go for UNIX type distros.

Reply:
I missed the make install-user part in the Makfile.am, perfect!